src/openms/source/FORMAT/ExperimentalDesignFile.cpp

```diff
@@ -185,6 +179,16 @@ namespace OpenMS
       // the file section
       std::map <String, Size> fs_column_header_to_index;
 
+      // Maps the sample number to all values of sample-related columns
+      std::map <unsigned, std::vector<String>> sample_content_map;
+      /*
```

Comment:
dead?

Reply:
Still thinking about implementing string support

Reply:
Ok could be removed!